webui/react/package.json

```diff
@@ -45,6 +45,7 @@
     "react-dom": "^17.0.2",
     "react-draggable": "^4.4.4",
     "react-helmet-async": "^1.1.2",
+    "react-hook-form": "^7.34.0",
```

Comment:
[antd Form](https://ant.design/components/form/) doesn't work in this case because it is rendered every time I hit a key. Good thing about react hooks form is that it only renders once so its efficient. However, I agree not to bring new libraries especially there is a similar library. 
I'll find other way without react hooks form.